### ISSUE TITLE
Bug: SP% equals to ¥NaN (NaN %)

### ISSUE THREAD

**Issue by Masedko** [PenguiniVogel/BuffUtility#41]
Today, it just stopped working.
![image](https://github.com/PenguiniVogel/BuffUtility/assets/78966998/e5edb4b5-97c9-4755-a03d-001fda69fa76)

**Comment by Orzeszu** [comment]
yea pls fix 🙏

**Comment by Masedko** [comment]
So, they removed from "data-asset-info" sell_reference_price. So, we need to use data-price from "sticker-name" div

**Comment by PenguiniVogel** [comment]
Yes I already fixed this locally about last month~ish, I'm just incredibly lazy not comitting things, I don't get paid for the amount of hours of my life I have to invest into this that I won't get back so cut me some slack.

**Comment by Masedko** [comment]
I know it's rude to ask again, but can you add a sp fix to the Chrome release?
